src/engine-actions/withdraw-funds.js

```diff
@@ -0,0 +1,25 @@
+const {
+  sendCoins
+} = require('../lnd-actions')
+
+/**
+ * Given a payment request, it pays the invoices and returns a refund invoice
+ *
+ * @param {String} addr wallet address to send the coins to
+ * @param {Integer} amount amount of coin to send to wallet address
+ * @param {Object} options
+ * @return {String} txid transaction for the withdrawal
+ */
+
+async function withdrawFunds (addr, amount) {
+  try {
+    const { txid } = await sendCoins(addr, amount, { client: this.client })
+    this.logger.debug('Coins successfully sent', { txid })
```

Comment:
[nit] update this to be something like `Funds withdrawn successfully` to match the method name